assets/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,57 @@
+
+import React, {Component} from 'react';
+import { Button } from './Button';
+import { MenuItems } from './MenuItems';
+import './NavBar.css';
+
+class NavBar extends Component {
+
+    state = {

```

Comment:
Just a minor thing, for consistency's sake, move the state declaration into a constructor. And create a constructor please.